src/servant/tt_000/10E8.c

```diff
@@ -450,12 +448,7 @@ void ServantInit(InitializeMode mode) {
     SpriteParts** spriteBanks;
     Entity* e;
 
-#ifdef VERSION_PC
-    // i exceeds the size of D_80170448
-    const int len = LEN(D_80170448);
```

Comment:
Why was this removed? This is an out of bounds bug

Reply:
I removed it because if it's there it causes the macOS and linux builds to fail.  I'll add it back in.  Maybe you can help me figure out how to fix

Reply:
I figured it out.  I had removed an extern variable declaration which was what was breaking those builds.